### ISSUE TITLE
Only show score only after submit. 

### ISSUE THREAD

**Issue by kwibus** [h5p/h5p-interactive-book#133]
If you want to use interactive book for a test. You don't want it to show score before you submit  your answers. 
So people can't change there answer until they get full score. 

If it would only show score after submit:  
* You still have to make sure the content you use insides interactive book does not show the answer. or don't allow you to retry.
* and people can cheat on a test in other ways. 

But it would be usable for in a test. 

I am thinking about making a merge request for this. But it would be nice to know if you think this feature is a good idee?

**Comment by otacke** [comment]
@kwibus H5P currently is not fit for high-stakes exams anyway as the evaluation takes place in the browser, not server-side.

**Comment by kwibus** [comment]
I know that H5P in processed client side. So answers are implicitly send to the user. 
And if they know some javascript. they can run 
```
H5P.instances[0].showSolutions()
```
And if you give people access to a computer with internet while taking exam. It becomes hard to prevent cheating anyway.

But that does not mean people always cheat.

But I think it's often to tempting and easy, for lots of students to change there answer, when  you show them there answer is wrong. 

That is a risk you have to consider when you create a test.
And I think there are use case were this is a acceptable risk, if the test is not to high-stakes


But I understand you reasoning. Maybe you don't want to give the impression that `interactive book` could be used for test.

But that leave me still with the question:  Would you consider accepting a pull request if i would take the time to write one?

**Comment by otacke** [comment]
@kwibus I can't answer your question, because I am not part of the H5P core team.
